huaweicloud/resource_huaweicloud_sfs_turbo.go

```diff
@@ -308,9 +391,10 @@ func resourceSFSTurboRead(_ context.Context, d *schema.ResourceData, meta interf
 	return nil
 }
 
-func buildTurboUpdateOpts(newSize int, isPrePaid bool) shares.ExpandOpts {
+func buildTurboUpdateOpts(newSize int, isPrePaid bool, hpcCacheBandwidth int) shares.ExpandOpts {
```

Comment:
it's better to put parameters with the same type together

```suggestion
func buildTurboUpdateOpts(newSize, hpcCacheBandwidth  int, isPrePaid bool) shares.ExpandOpts {
```

Reply:
OK